aksel.nav.no/website/e2e/smoketest.test.ts

```diff
@@ -23,4 +25,27 @@ test.describe("Smoketest all pages", () => {
       }
     });
   }
+
+  test.describe("sandbox examples", () => {
+    const folders = getDirectories("eksempler");
+
+    for (const folder of folders) {
+      const files = parseCodeFiles(folder.path, "eksempler");
```

Comment:
Bør hente ut `parseCodeFiles(folder.path, "templates");` også her for å teste malene